docs/src/modules/operations/pages/projects/broker-azure-eventhubs.adoc

```diff
@@ -0,0 +1,82 @@
+
+== Setting up the Event Hub
+
+Follow the Azure documentation to create a resource group, a namespace and a hub: https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create[Azure Event Hubs Quickstart]
+
+Make sure to define a number of partitions of at least the highest number of service instances you expect your Akka
+service to scale out to. For example the default upper limit with Akka is 10 instances.
+
+=== Shared Access Policy for the hub
+
+Create a shared access policy (SAS) for the "namespace" or the individual hub, if the individual hub is used that is the only hub that can be accessed from the services. With the namespace all hubs created in it can be accessed.
+
+Note the connection string `primary key`, the string should start with `Endpoint`:
+
+image::operations:azure-eventhubs-sas.png[Azure Event Hub SAS primary key]
+
+=== Set up the check point store
+
+Azure Blob Storage is used for checkpoint storage of consumers so needs to be set up as well as the event hub itself.
+
+Create a storage account, a blob container and set up a "connection string" as described in https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-java-get-started-send?tabs=passwordless%2Croles-azure-portal#create-an-azure-storage-and-a-blob-container[the Azure Event Hub Docs about checkpoint store]
+
+Note the account name and the container name you created.
+
+Under "Shared access tokens" for the created container, generate and note the Blob SAS Token:
+
+image::operations:azure-checkpoint-store-sas-token.png[Azure Event Hub Checkpoint Store SAS token]
+
+== Set up Akka access
+
+Store the event hub connection string and the checkpoint store SAS token as Akka secrets:
+
+----
+akka secret create generic azure-eventhub-secret \
+--literal connection-string="<hub connection string>" \
+--literal store-sas-token="<store SAS token>"
+----
+
+Configure the broker:
+
+----
+akka projects config set broker \
+--broker-service eventhubs \
+--event-hubs-connection-string-secret azure-eventhub-secret/connection-string \
+--event-hubs-checkpoint-store-sas-token-secret azure-eventhub-secret/store-sas-token \
+--event-hubs-checkpoint-store-endpoint "https://<your-storage-account-name>.blob.core.windows.net" \
+--event-hubs-checkpoint-store-container-name <your-container-name>
+----
+
+
+== Special caveats when using event hubs as broker
+
+=== Topic
+
+In Akka terminology, the event hub name is the topic name. A component subscribing or publishing to a topic is equivalent with it subscribing or publishing to an event hub.
+
+=== Retention
+Events always have retention, if they for some reason are not consumed within the retention time they are lost. This could be for example if pausing a consuming service.
+
+=== Ordering
+
+The ordering guarantee, that consumers see events in the order they were produced is per partition.
+
+Cloud event subject metadata entry, if present, is used as partition key for events produced by an Akka service.
```

Comment:
we might need example code of how to set this when publishing (same for the other brokers, if we don't have that already)